package.json

```diff
@@ -102,6 +102,7 @@
     "sortablejs": "^1.4.2",
     "source-sans-pro": "^2.0.10",
     "spin.js": "^2.3.2",
+    "string_score": "^0.1.22",
```

Comment:
Added after discussing with @jdesboeufs, I wasn't aware we already have two!
Which one is more appropriated here?